services/121-service/test/program/delete-program.test.ts

```diff
@@ -0,0 +1,78 @@
+import { HttpStatus } from '@nestjs/common';
+
+import { SeedScript } from '@121-service/src/scripts/seed-script.enum';
+import {
+  deleteProgram,
+  getProgram,
+  startCbeValidationProcess,
+} from '@121-service/test/helpers/program.helper';
+import { seedPaidRegistrations } from '@121-service/test/helpers/registration.helper';
+import {
+  getAccessToken,
+  resetDB,
+} from '@121-service/test/helpers/utility.helper';
+import {
+  programIdOCW,
+  programIdPV,
+  registrationCbe,
+  registrationsOCW,
+  registrationsPV,
+} from '@121-service/test/registrations/pagination/pagination-data';
+
+describe('Delete program', () => {
+  let accessToken: string;
+
+  it('should delete a nlrc programs', async () => {
+    // Arrange
+    await resetDB(SeedScript.nlrcMultiple);
```

Comment:
why not use nlrcMultipleSeecMock? I guess it doesn't matter much..

Reply:
Yes, I thought this was easier to read in an API tests. You need less context to understand what happens, but I don't really have a strong opinion about it

Reply:
fine